### ISSUE TITLE
Possible bug in signature check

### ISSUE THREAD

**Issue by nfriedly** [memcachier/memjs#173]
I'm not certain, but I think you intended to have a `typeof` here:

https://github.com/memcachier/memjs/blob/f10b26006e1a40394dc192f9edd628c0ec43189a/lib/memjs/memjs.js#L416-L417

```diff
-  if(callback === undefined && options !== 'function') {
+  if(callback === undefined && typeof options !== 'function') {
```

**Comment by saschat** [comment]
You are right. That is a bug. Will fix this shortly.

**Comment by saschat** [comment]
Fixed in version 1.3.2.
